plugins/modules/cloud/lxd/lxd_virtual_machine.py

```diff
@@ -0,0 +1,744 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2016, Hiroaki Nakamura <hnakamur@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: lxd_virtual_machine
+short_description: Manage LXD Virtual machines
+description:
+  - Management of LXD virtual machines
```

Comment:
```suggestion
  - Management of LXD virtual machines.
```
Descriptions (except short_description) should always have complete sentences in their paragraphs.